src/kubelet/kubelet_test.go

```diff
@@ -85,19 +79,99 @@ func TestScraper(t *testing.T) {
 	}
 }
 
-func ExcludeMissingMetricsPendingPod(_ string, spec *definition.Spec, ent *integration.Entity) bool {
-	metricsToExcludeForPendingPods := []string{"memoryUsedBytes", "memoryWorkingSetBytes", "cpuUsedCores",
+// kubeletExclusions is a helper that returns all the exclusions needed to assert the kubelet metrics without getting
+// false negatives.
+func kubeletExclusions() []exclude.Func {
+	// Network metrics are known to be missing on some environments.
+	networkMetrics := []string{"net.rxBytesPerSecond", "net.txBytesPerSecond", "net.errorsPerSecond"}
+
+	// TODO: Unclear why we need to exclude node utilization metrics.
+	nodeUtilizationMetrics := []string{"allocatableCpuCoresUtilization", "allocatableMemoryUtilization"}
```

Comment:
It is not unclear anymore, I would open an issue and link it in the TODO

Reply:
I need to doublecheck if node metrics are also affected by this, or did you check it already? I've only confirmed the bug for containers/pods.

Reply:
I guess they are since they are based as well on computed ones